lib/cronitor/config.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Cronitor
+  class << self
+    attr_accessor :api_key, :api_version, :environment,
+                  :logger, :config, :_headers,
+                  :TYPE_JOB, :TYPE_EVENT, :TYPE_SYNTHETIC, :MONITOR_TYPES, :YAML_KEYS
```

Comment:
@jeffbyrnes i know i'm doing something wrong here. I just need to define these constants on the module. 

Originally I was doing

```
module Cronitor
  TYPE_JOB = 'job'
...
```

But was getting a no method error when trying to access the constant.
```
NoMethodError:
       undefined method `YAML_KEYS' for Cronitor:Module
```